c/src/point_one/polaris/polaris.c

```diff
@@ -16,26 +16,31 @@
 #define MAKE_STR(x) #x
 #define STR(x) MAKE_STR(x)
 
+#define Print(x, ...) \
+  P1_fprintf(stderr, "polaris.c:" STR(__LINE__) "] " x, ##__VA_ARGS__)
+#define PrintError(x, ...) \
+  P1_perror("polaris.c:" STR(__LINE__) "] " x, ##__VA_ARGS__)
+
 #if defined(POLARIS_DEBUG) || defined(POLARIS_TRACE)
-# define DebugPrintf(x, ...) P1_printf(x, ##__VA_ARGS__)
+# define DebugPrint(x, ...) Print(x, ##__VA_ARGS__)
```

Comment:
minor style thing but I prefer these macros be more specific to us (i.e. P1Print or something) to avoid possible #def conflicts with other libraries. Granted this is a c file, so it could only come upstream, but still.

Reply:
Sure, I can do that. I thought about doing that and then didn't but it makes sense.